phaser-react/src/states/flow1/EnoughMoney.jsx

```diff
@@ -3,8 +3,9 @@
 export default function EnoughMoney({ send }) {
   return (
     <div>
-        EnoughMoney
-        <button onClick={() => send({type: "START"})}>continue</button>
+        <h1> Enough Money </h1>
+        <p> You managed to gather enough money to pay the coyote&#39;s price.</p>
```

Comment:
This is the last place I noticed the random characters replacing an apostrophe (').